config_search.py

```diff
@@ -1,15 +1,19 @@
 import os, sys, inspect
 
 # Full-text search config parameters
-
-SOLR_URL = {"prod": "http://localhost:8081/solr/core-1x/",
-            "dev": "http://localhost:8983/solr/#/core-x/"}
+SOLR_URL = {
+    "prod": "http://localhost:8081/solr/core-1x/",
```

Comment:
I guess the idea here is that this would be using ssh tunneling to reach production; if we were running it in kubernetes we may want to be able to override this via environment variable. No change needed now, just pointing it out for the future.

Reply:
yes, that is the idea